File: python/liczby23.cpp
/*
 * liczby23.cpp
 */


#include <iostream>
using namespace std;

int liczby1() {
    int ile = 0;
    for (int i = 1; i < 10; i++) {
        for (int j = 1; j < 10; j++) {
            if (i != j) {
                cout << i << j <<" ";
            }
        }
    } 
    return ile;
}

int main(int argc, char **argv)
{cout << "Liczb 2-cyfrowych:"; << liczby2();
	
	return 0;
}

File: python/liczby23.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  liczby23.py

def Liczby2():
    """
    Funkcja drukuje wszystkie liczby 2 cyfrowe, w których nie powtarzają
    się cyfry. Na koniec zwraca ilość takich liczb.
    Wykluczone liczby 11, 22, 33, ...
    """
    ile = 0
    for i in range(1, 10):
        for j in range(10):
            if i != j:
                print("{}{} ".format(i, j), end='')
                ile = ile + 1
    return ile
    
def Liczby3():
    
    ile = 0
    for i in range(1, 10):
        for j in range(10):
            for k in range(10):
                if i != j and i != k and j != k:
                    print("{}{}{} ".format(i, j, k), end='')
                    ile = ile + 1
    return ile

def main(args):
    print("Liczb 2-cyfrowych:", Liczby2())
    print("Liczb 3-cyfrowych:", Liczby3())
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
